ruby_programming/testing_with_rspec/introduction_to_rspec.md

```diff
@@ -0,0 +1,236 @@
+# Introduction to RSpec
```

Comment:
Can we remove this heading please? this will be generated on the site automatically 🙂

Reply:
:+1: